packages/lib/src/form-fields/file-upload.wc.svelte

```diff
@@ -26,6 +27,14 @@
   export let id = '';
   export let name = '';
   export let required = false;
+  enum cropOptions {
+    disabled = 0,
+    optional = 1,
+    mandatory = 2
+  }
```

Comment:
It would be better if the cropOptions attribute is defined by a set of options that are type string.